docs/changelog-template.md

```diff
@@ -0,0 +1,40 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+- Short functionality description #1
+- [FIX] Bug fix description #2
+
+## [2.0.5] - 2019-10-17
+
+### Added
+- Short functionality description #1
```

Comment:
Markdown linting will require that headers are separated from other content by a blank line